net/mtxclient/Portfile

```diff
@@ -3,23 +3,27 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        mujx mtxclient 2f519d2
-version             2018-07-14
-revision            1
+github.setup        Nheko-Reborn mtxclient 0.3.1 v
+epoch               1
 categories          net chat devel
 platforms           darwin
 license             GPL-3
 maintainers         {@scarface-one disroot.org:scarface} openmaintainer
 description         Client API for Matrix
 long_description    Client API library for the Matrix protocol, built on top of Boost.Asio.
 
-checksums           rmd160  28330b8f5c028792aaa5a1dbdd3738aea4854e7d \
-                    sha256  760a03fabae4cca250a5a7aa1328e7fb922186b431c1b65530ad083dcb9fda4c \
-                    size    92272
+checksums           rmd160  31722fa1af22f239c68527b4d677b79c06c748ea \
+                    sha256  4742e95097a5ec1be31685398173d9bf248ecc2fef96932bad8fe528de7255a2 \
+                    size    512153
 
 compiler.cxx_standard \
-                    2014
+                    2017
+# Need <variant> header
+platform darwin 17 {
+    compiler.blacklist-append clang
+}
```

Comment:
I would change the comment here to `Need <variant> header and std::optional::value`. The reason for the approach here is so that `std::optional::value` is available, although it happens to be sufficient for getting the `<variant>` header, since macOS 10.14 and later only run Xcode 10 and later.